packages/office-ui-fabric-react/etc/office-ui-fabric-react.api.ts

```diff
@@ -10959,12 +10959,9 @@ export function merge<T = {}>(target: Partial<T>, ...args: (Partial<T> | null |
 // @public
 export function mergeAriaAttributeValues(...ariaAttributes: (string | undefined)[]): string | undefined;
 
-// @public
+// @public (undocumented)
 export function mergeCustomizations(props: ICustomizerProps, parentContext: ICustomizerContext): ICustomizerContext;
 
-// @public
-export function mergeSettings(oldSettings?: Settings, newSettings?: Settings | SettingsFunction): Settings;
-

```

Comment:
There shouldn't be ANY changes to this file in this PR.. and this conflicts with master (which has the correct version.)

Reply:
I'm guessing this is why CI is failing. 😄

Reply:
Yea I was confused by this as well. I'm interested to know how `master` got in this state and how we can avoid it going forward. I'll reach out in the morning to see how I can help.

Reply:
`master` is in the state it should be, it's the PR that's out of sync. 😄 I'm confused as to how these changes crept into the PR (or didn't get merged from master correctly.)

Reply:
Thanks @JasonGore, I was able to resolve this :).